Sources/ViewControllers/Settings/CreditsViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  CreditsViewController.swift
+//  OpenFoodFacts
+//
+//  Created by Егор on 2/16/18.
+//  Copyright © 2018 Andrés Pizá Bückmann. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class CreditsViewController: UIViewController {
+
+    private var webView: UIWebView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupWebView()
+    }
+
+    private func setupWebView() {
+        webView = UIWebView(frame: self.view.frame)
+        webView.backgroundColor = .white
+        webView.clipsToBounds = true
+        webView.delegate = self
+        webView.scrollView.indicatorStyle = .white
+        self.view.addSubview(webView)
+        guard let htmlCreditsFilePath = Bundle.main.path(forResource: "Credits", ofType: "html"),
+            let htmlContent = try? String(contentsOfFile: htmlCreditsFilePath) else {
+                return
+        }
+        webView.loadHTMLString(htmlContent, baseURL: Bundle.main.bundleURL)
+    }
+
+}
+
+extension CreditsViewController: UIWebViewDelegate {
+
+    func webView(_ webView: UIWebView, shouldStartLoadWith request: URLRequest, navigationType: UIWebViewNavigationType) -> Bool {
+        guard let requestURL = request.url else {
```

Comment:
This logic, as it is, it's generating an error that says `This app is not allowed to query for scheme file` due to the `canOpenURL` call. This only happens once, when loading the initial html file. Please, change the method's implementation to the following to avoid this error:

```
if navigationType == .linkClicked, let requestURL = request.url {
    UIApplication.shared.openURL(requestURL)
    return false
}

return true
```